Makefile

```diff
@@ -124,8 +124,11 @@ E2E_REGISTRY_NAME := docker-registry
 E2E_REGISTRY_NAMESPACE := operator-controller-e2e
 
 export REG_PKG_NAME := registry-operator
-export REGISTRY_ROOT := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000
-export CATALOG_IMG := $(REGISTRY_ROOT)/e2e/test-catalog:e2e
+export LOCAL_REGISTRY_HOST := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000
+export CLUSTER_REGISTRY_HOST := localhost:30000
+export E2E_TEST_CATALOG_V1 := e2e/test-catalog:v1
```

Comment:
I'm seeing a lot of os.Getenvs in the e2es. Could we (if it's not there already), group the env vars that the e2e expects and add a comment? I know we'll find out relatively quickly if we deleted something we shouldn't, but I just want to avoid the CI run for that if we can.